### DISCUSSION TITLE
assets version

### DISCUSSION THREAD

**Discussion by behvandi11** [wireui/wireui#397] [BUG]
Hello
Good time...
WireUi is continuously updated and I updated it often. after update I run optimize:clear command but some times also needs to a hard refresh by ctrl + F5 so it is no problem for me but for users it is!
can I import wireui asset in webpack ? if it is possible after every build creation a new version compiled and then cached assets do not cause problems
if can I fix this problem by another ways please guide me
thank you

**Comment by ph7jack** [comment]
@behvandi11 The wireui scrips have an id of the build to prevent old caches, but I think it's not working. I'll make a fix for it.
